Firmware/xyzcal.cpp

```diff
@@ -569,6 +573,8 @@ void __attribute__((noinline)) xyzcal_scan_pixels_32x32_Zhop(int16_t cx, int16_t
 			sm4_set_dir(X_AXIS, d);
             //@size=242
 			DBG(_n("%d\n"), 64 - (r * 2 + d)); ///< to keep OctoPrint connection alive
+			lcd_set_cursor(4,3);
+			lcd_printf_P(PSTR("Countdown: %d "),64 - (r * 2 + d)); ////MSG_COUNTDOWN c=14
```

Comment:
Why not translated?

Reply:
Try to avoid translations on rare used messages to save space.